src/main/java/br/com/moip/api/RefundAPI.java

```diff
@@ -24,4 +29,9 @@ public Refund get(String refundId) {
         return client.get(String.format("/v2/refunds/%s", refundId), Refund.class);
     }
 
+    public RefundsListResponse list(String id) {
+        if (id.startsWith("PAY-")) return client.get(String.format("/v2/payments/%s/refunds", id), RefundsListResponse.class);
+
+        return client.get(String.format("/v2/orders/%s/refunds", id), RefundsListResponse.class);
```

Comment:
Como tem vários lugares que usam essas strings de endpoints, talvez você possa criar uma classe pra guardar essas constantes:
```
public class Endpoint {
       private static String REFUNDS = "/v2/orders/%s/refunds"

       public String buildRefundEndpointFor(String orderId) {
             return String.format(REFUNDS, id)  
       }
} 
```

Dai nesse RefundAPI ficaria assim:
`client.get(buildRefundEndpointFor(id), RefundsListResponse.class ) `

https://www.javaworld.com/article/2076481/learn-java/use-constant-types-for-safer-and-cleaner-code.html

Reply:
Feito! As demais classes entrarão neste padrão em um futuro release.